crates/sui-framework/src/lib.rs

```diff
@@ -164,7 +164,8 @@ fn verify_framework_version(pkg: &CompiledPackage) -> SuiResult<()> {
 
     if dep_framework != framework {
         return Err(SuiError::ModuleVerificationFailure {
-            error: "Sui framework version mismatch between package dependencies and Sui binary"
+            error: "Sui framework version mismatch detected.\
+                    Make sure that the sui command line tool and the Sui framework code used as a dependency correspond to the same git commit"
```

Comment:
Does this fit on one line (in Rust)? if not a line break would be nice, just rendered a bit weirdly on my screen currently (but might just be a device issue)

Reply:
Yeah, it is a little wide, but I was not sure if it's better to leave it or try to break it as I was not sure what the most natural way would be. Here's one try.